### PR TITLE
[9.0][REF]purchase_request_to_rfq: use separate method for reception message

### PR DESCRIPTION
**Pull request by zakiuu**
Separate the logic of sending reception message from the `do_transfer` method to ease the inheritance and the override of that part of code

### CODE REVIEW

File: purchase_request_to_rfq/models/stock.py [hunk 1]
```diff
@@ -58,3 +57,9 @@ def do_transfer(self):
                     self._purchase_request_picking_confirm_message_content(
                         picking, request, requests_dict[request_id])
                 request.message_post(body=message, subtype='mail.mt_comment')
+
+    @api.multi
+    def do_transfer(self):
+        res = super(StockPicking, self).do_transfer()
+        self._purchase_request_reception_message()
```
Comment (lmignon) [inline purchase_request_to_rfq/models/stock.py]: @zakiuu I would prefer a more meaningful name for the method. May be `_post_purchase_request_picking_confirm_message`

**Review by lmignon — CHANGES_REQUESTED** [review]

File: purchase_request_to_rfq/models/stock.py (hunk 1 above)
Comment (zakiuu) [reply purchase_request_to_rfq/models/stock.py]: @lmignon thanks for the review, done!

**Review by lmignon — APPROVED** [review]
@zakiuu Thank you! (code review)

**Review by ThomasBinsfeld — APPROVED** [review]

**Review by JordiBForgeFlow — APPROVED** [review]
+1 code review

**Comment by JordiBForgeFlow** [comment]
@lreficent can you review?

**Review by andhit-r — APPROVED** [review]

**Review by LoisRForgeFlow — APPROVED** [review]
LGTM :+1:

**Comment by zakiuu** [comment]
@jbeficent is it possible to merge this one?

**Review by gurneyalex — APPROVED** [review]

**Pull request merged**